### PR TITLE
Navbar component

### PR DESCRIPTION
**Pull request by Othmanosx**
Used material ui to create Navbar component and added responsive design for mobile devices and custom font from google fonts

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    {
+      label: "Contact us",
+      href: "/contact"
+    }
+  ];
+  
+  const useStyles = makeStyles(() => ({
+    header: {
+      backgroundColor: "#3AAFA9",
+    //   paddingRight: "79px",
```
Comment (Ravyar19) [inline src/components/Navbar/Navbar.jsx:50]: Please do not leave comments

**Review by Ravyar19 — CHANGES_REQUESTED** [review]
Delete Comments

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import logo from './logo.svg';
 import './App.css';
+import Navbar from './components/Navbar/Navbar.jsx'
 
 function App() {
   return (
     <div className="App">
+      <Navbar />
```
Comment (maqalaqil) [inline src/App.js:8]: remove this

**Review by maqalaqil — CHANGES_REQUESTED** [review]
LGTM just change app.js @Othmanosx amazing work man

**Review by murtaja1 — APPROVED** [review]

**Pull request merged**